### PR TITLE
Milestone #1: List structured implemented

### PR DESCRIPTION
**Pull request by lordksix**
In this PR, the changes are as follows:
- Add App UI (HTML and CSS).
- Add dynamic rendering of list items from a hardcoded array of objects.

 Array of Object
https://github.com/lordksix/Minimalistic-ToDoList-App/blob/f6122feb90ac6fc09cf934fb5b972935ecc8acf7/src/index.js#L5-L11
Function to render each list element
https://github.com/lordksix/Minimalistic-ToDoList-App/blob/f6122feb90ac6fc09cf934fb5b972935ecc8acf7/src/index.js#L24-L29
Rendering of elements
https://github.com/lordksix/Minimalistic-ToDoList-App/blob/f6122feb90ac6fc09cf934fb5b972935ecc8acf7/src/index.js#L45-L47
- Add Class for List elements
https://github.com/lordksix/Minimalistic-ToDoList-App/blob/f6122feb90ac6fc09cf934fb5b972935ecc8acf7/src/modules/ListItemMod.js#L36-L42
- Add item completed functionality
https://github.com/lordksix/Minimalistic-ToDoList-App/blob/f6122feb90ac6fc09cf934fb5b972935ecc8acf7/src/index.js#L31-L43

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -1 +1,49 @@
...
+import './index.css';
+import svg from './asset/resource/icons.svg';
+
+const item1 = new ListItem('Item1', 1);
+const item2 = new ListItem('Item2', 2, true);
+const item3 = new ListItem('Item3', 3);
+const item4 = new ListItem('Item4', 4, true);
+const item5 = new ListItem('Item5', 5);
+
+const arrayItems = [item1, item2, item3, item4, item5];
```
Comment (brytebee) [inline src/index.js]: Kindly render the list items in the order of their indexes.

Consider the code snippet below:

```
const arrayItems = [item1, item2, item3, item4, item5].sort((a, b) => a.index - b.index);
```

**Review by brytebee — CHANGES_REQUESTED** [review]
Hi, @lordksix  👋,

##

### Highlights💡
- Good commit message.✔️ 
- Good PR Summary.✔️
- Great UI design.✔️
- App functions as it should.✔️

##

Good job so far coming so far with the progress you've made! 💪💪
 
However, there are some issues that you still need to work on to go to the next project but you are almost there!



# Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Please, remember to tag me in your question so I can receive the notification.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @brytebee in your question so I can receive the notification.
### _**Use @brytebee on slack for faster responses.**_


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/index.js (hunk 1 above)
Comment (lordksix) [reply src/index.js]: Thank you for the suggestion. Added throught the commit 6f2515b87debe71dc30400b6a855d314d09452cb

**Review by alexander16108 — APPROVED** [review]
Hi @lordksix , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**